src/core/Akka.Persistence.TestKit/JournalInterceptors.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace Akka.Persistence.TestKit
+{
+    using System;
+    using System.Threading.Tasks;
+
+    internal static class JournalInterceptors
```

Comment:
Overall, I think the set of these are elegantly simple - I like the look of it and I like the idea that they're extensible in the future. I think the design is good @valdisz